### PR TITLE
NumberBox: Change StepFrequency to SmallChange and LargeChange

### PR DESCRIPTION
**Pull request by jevansaks**
We got feedback that the "StepFrequency" name was confusing and also that folks wanted support for PageUp/PageDown to make large changes. This change adds SmallChange and LargeChange properties and gets rid of StepFrequency.

### CODE REVIEW

File: dev/NumberBox/NumberBox.cpp [hunk 1]
```diff
@@ -367,12 +367,22 @@ void NumberBox::OnNumberBoxKeyDown(winrt::IInspectable const& sender, winrt::Key
     switch (args.OriginalKey())
     {
         case winrt::VirtualKey::Up:
-            StepValueUp();
+            StepValue(-SmallChange());
```
Comment (StephenLPeters) [inline dev/NumberBox/NumberBox.cpp]: >-SmallChange() [](start = 22, length = 14)

Up goes down?

File: dev/NumberBox/InteractionTests/NumberBoxTests.cs [hunk 2]
```diff
@@ -64,15 +64,24 @@ public void UpDownTest()
...
                 Log.Comment("Verify that up button increases value by 5");
                 upButton.InvokeAndWait();
                 Verify.AreEqual(5, numBox.Value);
 
+                Log.Comment("Change LargeChange value to 50");
+                RangeValueSpinner largeChangeNumBox = FindElement.ByName<RangeValueSpinner>("LargeChangeNumberBox");
+                largeChangeNumBox.SetValue(50);
+                Wait.ForIdle();
+
+                Log.Comment("Verify that up button increases value by 50");
```
Comment (StephenLPeters) [inline dev/NumberBox/InteractionTests/NumberBoxTests.cs]: >50 [](start = 70, length = 2)

5

File: dev/NumberBox/NumberBox.cpp (hunk 1 above)
Comment (jevansaks) [reply dev/NumberBox/NumberBox.cpp]: Oops, good catch. I hadn't pushed all my local changes.

File: dev/NumberBox/InteractionTests/NumberBoxTests.cs [hunk 3]
```diff
@@ -64,15 +64,24 @@ public void UpDownTest()
...
+                upButton.InvokeAndWait();
```
Comment (StephenLPeters) [inline dev/NumberBox/InteractionTests/NumberBoxTests.cs]: The up button always does small increase doesn't it?

Comment (jevansaks) [reply dev/NumberBox/InteractionTests/NumberBoxTests.cs]: Yes, the comment was wrong. i fixed the test so it sends PageUp to test the large change behavior.

**Comment by StephenLPeters** [comment]
Should we ensure that largeChange > smallChange

**Comment by jevansaks** [comment]
> Should we ensure that largeChange > smallChange

+@SavoySchuler for visibility but I would say no. In general i don't like enforcing things that don't strictly need to be enforced. Throwing exceptions is rude and coercing values in this case seems unnecessary.

**Pull request merged**